pkg/projects/Microsoft.NETCore.DotNetHost/Microsoft.NETCore.DotNetHostResolver.pkgproj

```diff
@@ -16,6 +16,9 @@
     <Dependency Include="Microsoft.NETCore.DotNetHost">
       <Version>$(HostFullVersion)</Version>
     </Dependency>
+    <Dependency Include="Microsoft.NETCore.DotNetAppHost">
```

Comment:
@schellap Can you please an issue that tracks all chicken-and-egg issues you have in this PR so that we get them cleaned up?

CC @piotrpMSFT @livarcocc